auacm/angular/src/app/pipes/iif.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'iif'
+})
+export class IifPipe implements PipeTransform {
```

Comment:
This may no longer be needed (from my limited experience with Angular 2, you can use the ternary operator inside HTML now).